apicast/src/proxy.lua

```diff
@@ -361,6 +362,15 @@ function _M:access(service)
 
   local credentials, err = service:extract_credentials()
 
+  if keycloak then
```

Comment:
Only when the method is oauth, right? Otherwise this would be happening for user_key too.


Also this might belong to the `service:extract_credentials` method somehow. Not sure how yet though.